solidity/ecdsa/contracts/libraries/Wallets.sol

```diff
@@ -32,26 +32,12 @@ library Wallets {
         bytes32 digestToSign;
     }
 
-    struct Signature {
-        uint8 v;
-        bytes32 r;
-        bytes32 s;
-    }
-
     struct Data {
         // Mapping of keccak256 hashes of wallet public keys to wallet details.
         // Hash of public key is considered an unique wallet identifier (walletID).
         mapping(bytes32 => Wallet) registry;
     }
 
-    event SignatureRequested(bytes32 indexed walletID, bytes32 indexed digest);
-
-    event SignatureSubmitted(
```

Comment:
We have the same even in WalletRegistry which we also need to remove.

Reply:
d57285be715557c96ed49015dfe8ad31e3e71c83